tests/runtests.py

```diff
@@ -233,6 +234,22 @@ def actual_test_processes(parallel):
         return parallel
 
 
+def validate_selenium(selenium_args):
+    """
+    Validate the browsers requested and raise appropriate exception.
+    """
+    # Remove any browser repition.
+    browsers = list(set(selenium_args.split(',')))
+    for browser in browsers:
+        try:
+            # Create a class instance to see if everything is fine.
+            import_string('selenium.webdriver.%s.webdriver.WebDriver' % browser)().quit()
```

Comment:
Removing this would raise a browser error on each method, when an invalid browser is provided. Right now, it is being raised only once in the beginning.
I don't think removing this would resolve the Jenkins error though.